src/coreclr/vm/methodtable.inl

```diff
@@ -299,6 +299,13 @@ inline BOOL MethodTable::IsValueType()
     return GetFlag(enum_flag_Category_ValueType_Mask) == enum_flag_Category_ValueType;
 }
 
+inline BOOL MethodTable::IsContinuation()
+{
+    LIMITED_METHOD_DAC_CONTRACT;
+
+    return m_pParentMethodTable == CoreLibBinder::GetClassIfExist(CLASS__CONTINUATION);
```

Comment:
Probably cannot use `GetClassIfExist` here, it looks quite expensive to repeatedly call if the class hasn't been loaded. I suppose I can stash the type in a static when we create one of the continuation sub types.